src/sql/src/func.rs

```diff
@@ -2151,7 +2390,7 @@ lazy_static! {
                         pg_catalog.array_lower($1, 1),
                         pg_catalog.array_upper($1, 1),
                         1) as g(s)
-                "), 13112;
```

Comment:
this oid changed. was it wrong?

Reply:
Yeah; in PG 14:

```
select oid from pg_proc where proname = '_pg_expandarray';
  oid
-------
 13395
```

Reply:
This is a change from pg 13. We should perhaps make this function not ignored, and specify a specific postgres version it uses. Some day.